egl_preview.hpp

```diff
@@ -56,4 +58,5 @@ class EglPreview : public Preview
 	int y_;
 	int width_;
 	int height_;
+	std::string infoText;
```

Comment:
At the risk of being really annoying, libcamera-apps are generally using snake_case (just when we've started getting used to camelCase for libcamera itself), and members of (non-trivial) classes get a _ on the end...

Reply:
Oops, ill fix that.